requirements.txt

```diff
@@ -1,2 +1 @@
-daal==2021.4.0

```

Comment:
Where this does used?

Reply:
¯\_(ツ)_/¯

Reply:
I mean it's used as requirements for daal4py package/installation but it's meaningless since daal4py will be dependent on daal4py==2021.4.0 which is ignored by pip

Reply:
from setup.py:
```python
install_requires = []
with open('requirements.txt') as f:
    install_requires.extend(f.read().splitlines())
    if IS_MAC:
        for r in install_requires:
            if "dpcpp_cpp_rt" in r:
                install_requires.remove(r)
                break
```

Reply:
prod package deps are handled internally. So this file might be not needed at all?